community-tutorials/ruby-on-rails-in-production/01-en.md

```diff
@@ -0,0 +1,269 @@
+
+To adjust this we need to edit the following file:
+
+```sh
+sudo nano /etc/nginx/conf.d/mod-http-passenger.conf
+```
+
+There we change the line to the following:
+
+```diff
+- passenger_ruby /usr/bin/passenger_free_ruby;
++ passenger_ruby /home/USERNAME/.rbenv/shims/ruby;
+```
+
+The important thing is that we change `USERNAME` in the name of our user. To show what it is, we run `whoami`.
+
+Once we have saved the file, we execute the following command to start NGINX:
+
+```sh
+sudo service nginx start
+```
+
+If we now access the IP or address of the server via Webbrowser and receive the message "Welcome to NGINX", we have configured NGINX correctly.
+
+
+Now we can create a new configuration file by creating the following file with the editor nano:
+
+```sh
+sudo nano /etc/nginx/sites-enabled/rails_app
+```
+
+We copy the following content into it:
+
+```
+server {
+  listen 80;
+  listen [::]:80;
+
+  server_name _;
+  root /var/www/rails_app/public;
+
+  passenger_enabled on;
+  passenger_app_env production;
+
+  location /cable {
+    passenger_app_group_name myapp_websocket;
+    passenger_force_max_concurrent_requests_per_process 0;
+  }
+
+  client_max_body_size 10m;
+
+  location ~ ^/(assets|packs) {
+    expires max;
+    gzip_static on;
+  }
+}
+```
+
+You can modify the following details according to your needs:
+
+* server_name
+  * Instead of the underscore `_`, you can enter your domain here on which your Rails app should run. With underscore `_`, every domain that is routed to the server is served with the Rails app.
+* root
+  * `/var/www/rails_app` is the location of our Rails app. You can customise this, the important thing is that the path always ends with `/public`.
+* passenger_app_env
+  * If you prefer to run your Rails app for development in the development environment, you can specify `development` here instead of `production`.
+* location /cable
+  * If you are sure that your app will not use an action cable for websockets, you can remove this block from the file.
+* client_max_body_size
+  * If you want to allow your users to upload files to your Rails app, you can set a maximum upload file size here. In the default setting, this is 10 MB.
+* location ~ ^/(assets|packs)
+  * The assets and packs directories contain javascript and css files from your Rails app. To make these files more performant, they are cached and zipped with the block. You can extend this with more directories.
+
+
+Once you have saved the file, we change to the directory `/var/www/`. Now you can copy your existing Rails app into the folder `rails_app`. Alternatively, you can use tools like Capistrano for deployment.
+
+To install Rails, use the gem install command provided by RubyGems:
+
+```sh
+gem install rails
+```
+
+To verify that you have everything installed correctly, you should be able to run the following:
+
+```sh
+rails --version
+```
+
+If it says something like `Rails 7.0.2`, you are ready to continue.
```

Comment:
I think this is no problem, but I´m getting `Rails 6.1.3.1`

Reply:
Is not relevant which version, since this changes quickly I have removed the part.